admin-ui/plugins/auth-server/helper/constants.js

```diff
@@ -0,0 +1,63 @@
+// LDAP constants
+export const LDAP_READ = 'LDAP_READ'
+export const LDAP_WRITE = 'LDAP_WRITE'
+export const LDAP_DELETE = 'LDAP_DELETE'
+
+export const ACTIONS = {
+  EDIT_LDAP: 'authNLdap/editLdap',
+  ADD_LDAP: 'authNLdap/addLdap',
+}
+
+export const FEATURES = {
+  LDAP_EDIT: 'ldap_edit',
+}
```

Comment:
Consider feature tokens for delete/add if gated in UI

If edit is feature-gated, add tokens for delete/add to keep policy consistent.



```diff
 export const FEATURES = {
   LDAP_EDIT: 'ldap_edit',
+  LDAP_DELETE: 'ldap_delete',
+  LDAP_ADD: 'ldap_add',
 }
```